src/java.security.jgss/macosx/native/libosxkrb5/SCDynamicStoreConfig.m

```diff
@@ -51,7 +51,7 @@ void _SCDynamicStoreCallBack(SCDynamicStoreRef store, CFArrayRef changedKeys, vo
         jmethodID jm_Config_refresh = (*env)->GetStaticMethodID(env, jc_Config, "refresh", "()V");
         CHECK_NULL(jm_Config_refresh);
         (*env)->CallStaticVoidMethod(env, jc_Config, jm_Config_refresh);
-        if ((*env)->ExceptionOccurred(env) != NULL) {
```

Comment:
I believe we can keep the original behavior, because its being compared to `NULL` and not as a boolean.

> Returns the exception object that is currently in the process of being thrown, or NULL if no exception is currently being thrown.

Reply:
One concern pointed out by the umbrella bug is that ExceptionOccured(env) returns a jthrowable reference that should be freed by DeleteLocalRef(jthrowable). Even if the reference could be automatically freed later, it's probably still a good practice to either free them early or just not create them at all.